README.adoc

```diff
@@ -111,6 +111,9 @@ spring:
 In the example above, it `spring.cloud.kubernetes.config.namespace` had not been set,
 then the ConfigMap named `c1` would be looked up in the namespace that the application runs
 
+If a profile is active then it will also be looked at as `name-profile`. For example if `production` profile
+is active in the above example a `configMap` with name `c1-production` will fetched.
```

Comment:
This [README.adoc file was generated](https://github.com/spring-cloud/spring-cloud-kubernetes/blob/master/README.adoc#documentation). Your change will be lost when the file is generated again. You should change the [documentation source files](https://github.com/spring-cloud/spring-cloud-kubernetes/tree/master/docs/src/main/asciidoc) instead.